### PR TITLE
feat: localisation IT, EN, DE, FR, ES

### PR DESCRIPTION
**Pull request by marcouberti**
## Description

This PR add support for this languages: IT, EN, DE, FR, ES

## Checklist

- [X] I have followed the indications in the [CONTRIBUTING](../CONTRIBUTING.md).
- [X] The documentation related to the proposed change has been updated accordingly (plus comments in code).
- [X] I have successfully run tests with my changes locally.
- [X] It is ready for review! :rocket:

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1,223 +1,265 @@
...
-  ~ it under the terms of the GNU Affero General Public License as
-  ~ published by the Free Software Foundation, either version 3 of the
-  ~ License, or (at your option) any later version.
-  ~ This program is distributed in the hope that it will be useful,
-  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
-  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-  ~ GNU Affero General Public License for more details.
-  ~ You should have received a copy of the GNU Affero General Public License
-  ~ along with this program. If not, see <https://www.gnu.org/licenses/>.
-  -->
```
Comment (immuniopensource) [inline app/src/main/res/values/strings.xml:14]: Is this removal intended?

**Comment by ImmuniDGR** [comment]
🚔 **Safety Check** 🚔

🔰 **Result** 🔰
- The CircleCI configuration file has been modified.
- The revision of the scheduler submodule has changed.


⚠️ Some configuration files don't match the master branch. If you did not perform these changes, **please rebase on the master branch**.

🛠 **Diagnostic information** 🛠
- CircleCI scheduler pipeline: #498 (id: c6bf0e01-dcef-41e7-abe4-3721fafe056f)
- Last verified commit: 6ea41e101312ca09c4a453aaa8a26e87ce03d1e3
- Time of check: 28/05/2020, 17:30:59 UTC
- The following protected files have been checked for changes: .editorconfig, .gitmodules, Gemfile, Gemfile.lock, app/build.gradle, app/proguard-rules.pro, build.gradle, ci/changelog\_config.tpl, ci/danger/commitlint/index.ts, ci/danger/commitlint/rules.ts, ci/danger/danger.ts, ci/danger/ktlint.ts, dangerfile.ts, fastlane/Appfile, fastlane/Fastfile, gradle.properties, gradlew, gradlew.bat, package.json, publiccode.yml, scheduler\_config.json, settings.gradle, template.properties, yarn.lock.

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (marcouberti) [reply app/src/main/res/values/strings.xml:14]: No. Restoring it in all files.

**Comment by ImmuniDGR** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  :white_check_mark: ktlint passed
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>

  :white_check_mark: ktlint passed
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>

  :white_check_mark: ktlint passed
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>

  :white_check_mark: Commitlint passed
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>Thank you for submitting a pull request! The team will review your submission as soon as possible.</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 778508e720e94ae0d32e7e27bd3fd02759eebc4f
</p>
